substrate/frame/ranked-collective/src/lib.rs

```diff
@@ -1003,5 +1007,9 @@ pub mod pallet {
 		fn demote(who: &Self::AccountId) -> DispatchResult {
 			Self::do_demote_member(who.clone(), None)
 		}
+
+		fn max_rank() -> Self::Rank {
+			T::MaxRank::get() as Rank
```

Comment:
The pallet does not seem to use the value at all? So it does not have a mas-rank?  
Then this return here seems wrong.

Reply:
`max_rank()` is used in pallet-core-fellowship in-place of the previous hardcoded value of 9 https://github.com/paritytech/polkadot-sdk/blob/f34d8e3cf033e2a22a41b505c437972a5dc83d78/substrate/frame/core-fellowship/src/lib.rs#L153 to permit higher ranks if needed.

Reply:
Oh I see! In thinking of generalizing a way for other pallets that use the RankedMember trait.

I seem to have over engineered here. This function is indeed not needed.

Thanks for the review!

Just the Config in the core-fellowship pallet will do.